plugins/techdocs/config.d.ts

```diff
@@ -86,6 +86,12 @@ export interface Config {
                * @visibility secret
                */
               secretAccessKey: string;
+              /**
+               * ARN of role to be assumed
+               * attr: 'roleArn' - accepts a string value
+               * @visibility secret
+               */
+              roleArn: string;
```

Comment:
There is a detailed configuration reference for TechDocs which needs to be updated. :) https://backstage.io/docs/features/techdocs/configuration

Also, will you be able to add a small note about using `roleArn` at https://backstage.io/docs/features/techdocs/using-cloud-storage#configuring-aws-s3-bucket-with-techdocs (before the `4. That's it` part)? Pointing to some official AWS docs is totally fine!

Reply:
Yep, will do - I'll go ahead and point those back to the V2 JS sdk docs as well, since those don't seem to have been updated.